src/blockchain/database/common/BlockFilter.cpp

```diff
@@ -237,29 +228,23 @@ auto BlockFilter::store(
     const GCS& filter) const noexcept -> bool
 {
     try {
-        const auto proto = [&] {
-            auto out = proto::GCS{};
+        proto::GCS proto{};
+        if (!filter.Internal().Serialize(proto))
+            throw std::runtime_error{"Failed to serialize gcs"};
```

Comment:
He throws in a noexec function - I suggest you fix the signature as it is so patently wrong.